src/styles/calendar.scss

```diff
@@ -0,0 +1,63 @@
+$size-of-cell: 100px;
+$size-of-gap: 1px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  padding: 10px;
+  gap: $size-of-gap;
+  width: 7*$size-of-cell+6*$size-of-gap;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    background-color: #eee;
+    width: $size-of-cell;
+    height: $size-of-cell;
+    border: 1px solid black;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: 0.5s;
```

Comment:
Checklist: 
[STYLES] - Make sure to add transition style under general selector, not the one with :hover - this way transition will work smoothly both ways.
And don't forget to specify property which will be changed on hover